staking/integration-tests/src/utils/account.rs

```diff
@@ -58,3 +59,11 @@ pub fn fetch_account_data_bytemuck<T: Pod + Zeroable + AccountDeserialize>(
     let size = std::mem::size_of::<T>();
     T::try_deserialize(&mut &svm.get_account(account).unwrap().data.as_slice()[..size + 8]).unwrap()
 }
+
+pub fn fetch_positions_account(
```

Comment:
After merging with main the best place for this function is either in `solana/utils.rs` or `staking/utils.rs`

Reply:
It ended up in `solana/utils.rs`